splunk_eventgen/lib/eventgenoutput.py

```diff
@@ -22,11 +23,10 @@ def __init__(self, sample):
         self._setup_logging()
         self.output_counter = None
 
-
     def __str__(self):
         """Only used for debugging, outputs a pretty printed representation of this output"""
         # Eliminate recursive going back to parent
-        temp = dict([ (key, value) for (key, value) in self.__dict__.items() if key != '_c'])
+        # temp = dict([(key, value) for (key, value) in self.__dict__.items() if key != '_c'])
```

Comment:
would this actually be a functional change?

Reply:
No, I made this change manually because we don't currently use temp.